resources/metadata/whitesourceExecuteScan.yaml

```diff
@@ -255,7 +255,7 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
-        default: "https://github.com/SAP/SapMachine/releases/download/sapmachine-11.0.2/sapmachine-jre-11.0.2_linux-x64_bin.tar.gz"
+        default: "https://sap.github.io/SapMachine/latest/21/linux-x64/jre/"
```

Comment:
this does not look like a tar.gz file. Presumably the code will try and unpack whatever it fetches here. Did you test it?

Reply:
Please test this once (locally with `go run ...`) to make sure the redirect happens and the JRE 21 is fine to execute this mend scan.

Reply:
the given link is a HTTP '302 Temp Redirect' to the latest available release of SapMachine 21 linux-x64 JRE.
requested comment added to 
https://github.com/skateball/jenkins-library/blob/master/resources/metadata/whitesourceExecuteScan.yaml#L250
description:
          "URL used for downloading the Java Runtime Environment (JRE) required to run the WhiteSource Unified Agent. The URL is a HTTP 302 Temp Redirect to the always latest available SapMachine 21, linux-x64, JRE"